src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -466,7 +466,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     DependsOnTargets="_ExpandRuntimePackageRoot">
 
     <PropertyGroup>
-      <CompositeTargetDir>$(TargetDir)Composite\</CompositeTargetDir>
+      <CompositeTargetDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', 'Composite', '$(Configuration)')</CompositeTargetDir>
```

Comment:
There seems to be a missing second closing parenthesis corresponding to the one after the initial dollar sign, I believe that's why <code>NormalizeDirectory</code> doesn't get expanded and creation of the path fails the build.